src/categorical_algebra/CSetDataStructures.jl

```diff
@@ -857,4 +858,37 @@ function init_acset(T::Type{<:ACSet{CD,AD,Ts}},body) where {CD <: CatDesc, AD <:
   code
 end
 
+""" Dataframes backed by ACSets
+
+Useful as a "view" onto an ACSet, and also for computing limits
+"""
+
+""" P is a symbol, the object in the schema that this ASet focuses onto
```

Comment:
I see two docstrings here, I assume it's supposed to be one docstring?

Reply:
I was kind of thinking of the first docstring as the "section header", and the second docstring as the comment for the struct itself.

Also, do you have thoughts on the name `ACSetView`, and also whether this should be in a new file?

Reply:
I was thinking of this as a section header, and then a struct docstring.

On a related note, do you think that this should be a in a new file, and not in CSetDataStructures?

Reply:
Ah, I see. I am the kind of guy who thinks modules should have sections.

For just the views, I'd say keep it here since it's not too much code, but now that you're adding querying functionality in #345 maybe a new file would make sense?